src/app/api/quality_matrix/models.py

```diff
@@ -1,18 +1,38 @@
+from dataclasses import dataclass
 from enum import Enum
-from typing import Union
+from typing import Optional
 
 from pydantic import BaseModel, Field
 from sqlalchemy import JSON, Column, Integer, Text
 from sqlalchemy.orm import declarative_base
 
 
-class ColumnOutputModel(BaseModel):
+@dataclass(frozen=True)
+class QualityOutput:
+    metadatum: str
+    level: Optional[
+        int
+    ]  # Needs to be optional since timeline database does not have levels - yet
+    columns: dict[str, float]
+
+
+class QualityOutputModel(BaseModel):
```

Comment:
why duplicate it to a `QualityOutput`?

Reply:
I would also argue that `OutputResponse` is redundant :)

Reply:
One is a pydantic model, the other is a dataclass - I am currently mixing their usage, thus I need both. I could reduce it to one, but of course, this way we loose some functionality.